frontend/components/CalendarFull.vue

```diff
@@ -0,0 +1,472 @@
+<script setup lang="ts">
+import type { CalendarOptions } from "@fullcalendar/core";
+import dayGridPlugin from "@fullcalendar/daygrid";
+import interactionPlugin from "@fullcalendar/interaction";
+import timeGridPlugin from "@fullcalendar/timegrid";
+import FullCalendar from "@fullcalendar/vue3";
+import { useTolgee, useTranslate } from "@tolgee/vue";
+
+const fullCalendar = ref();
+const store = useActivityStore();
+const isActivityInfoVisible = ref(false);
+const activities = computed(() => store.activityData as Activity[]);
+const client = useSanctumClient();
+const alreadyAdded = ref(false);
+const { t } = useTranslate();
+const toast = useToast();
+const props = defineProps({
+    id: {
+        type: String,
+        required: true,
+    },
+});
+const activityId = ref("");
+const calendarId = ref("");
+const onlyShow = ref(true);
+const update = ref(false);
+const address = ref("");
+const cost = ref("");
+const created_at = ref("");
+const description = ref("");
+const email = ref("");
+const estimated_duration = ref("");
+const journey_id = ref("");
+const latitude = ref("");
+const longitude = ref("");
+const link = ref("");
+const mapbox_id = ref("");
+const name = ref("");
+const opening_hours = ref("");
+const phone = ref("");
+const updated_at = ref("");
+const tolgee = useTolgee(["language"]);
+
+interface EventObject {
+    event: Event;
+}
+
+interface Event {
+    _def: {
+        extendedProps: {
+            activity_id: string;
+            defId: string;
+        };
+        publicId: string;
+    };
+    _instance: {
+        range: {
+            start: {
+                toISOString: () => string;
+            };
+            end: {
+                toISOString: () => string;
+            };
+        };
+    };
+    remove: () => void;
+}
+
+interface CalendarActivity {
+    id: string;
+    title: string;
+    start: string;
+    end: string;
+    allDay: boolean;
+    activity_id: string;
+}
+
+const { data: currUser } = await useAsyncData("userRole", () =>
```

Comment:
Do we need to request this again. It is already requested in the general journey. Could be passed as prop like for ActivityPool or create a user store.